packages/release-please/src/release-please.ts

```diff
@@ -52,6 +53,7 @@ interface BranchOptions {
   bumpMinorPreMajor?: boolean;
   path?: string;
   changelogPath?: string;
+  manifest?: boolean;
```

Comment:
I'm not particularly a fan of this configuration. I would prefer that the manifest releaser be a `ReleaseType`, but that's not available in release-please -- you have to use a specific factory method. If we know that `manifest` will be a release type, then I'd prefer handling this by adding a `manifest` enum option to `releaseType` and special case handling it to use the new factory until we clean up how to build a manifest PR.